srcs/requirements/server/public/app.js

```
@@ -0,0 +1,67 @@
+ * @brief Loads a specified HTML page into the #app element
+ */
+function loadPage(page) {
+    return __awaiter(this, void 0, void 0, function* () {
+        const app = document.getElementById("app");
+        if (!app) {
+            console.log("Error: app not found");
+            return;
+        }
+        try {
+            // Fetch the page content
+            const response = yield fetch("pages/" + page);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)